pages/WhatIsOppia.html

```diff
@@ -8,7 +8,7 @@
 
   <p>
     Oppia is an online learning tool that enables anyone to easily create and
-    share interactive activities ('explorations') which try to recreate the
+    share interactive activities ("explorations") which try to recreate the
```

Comment:
I made a change as example on this page and forgot to reset it... Oh but this is a technical/new word, should it be bold instead of single-quoted ?